playwright-tests/workbox-broadcast-update/sw.js

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
Generally speaking, most of the `sw.js` files from the old integration tests can be moved over to the new test suite as-is.

The small changes that I've made are usually around setting explicit `cacheName` properties using `self.registration.scope`, which are guaranteed to be unique for each test execution, thanks to the `generateIntegrationURL()` helper funciton.

That way, each cache ends up being assigned a unique name and that helps avoid conflicts when running multiple tests in parallel.